qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -413,6 +414,7 @@ def submit(self):
         if self._future is not None or self._job_id is not None:
             raise JobError("We have already submitted the job!")
         self._future = self._executor.submit(self._submit_callback)
+        Publisher().publish("terra.job.started", self)
```

Comment:
Can you revise the name of the event? It seems we (slightly) favour the present tense (`.start`), and depending on the scope of the changes, it might be interesting to include something that explicitly mentions the provider in the name.

Reply:
Also, can you add an `Events:` section to the docstring?

Reply:
Ok, something like `ibmq.job.start` work for you?

Reply:
If it works for you, yeah, sounds good!